doc/man7/EVP_KDF-PVKKDF.pod

```diff
@@ -57,7 +57,7 @@ L<OSSL_PROVIDER-legacy(7)>
 
 =head1 HISTORY
 
-This functionality was added in OpenSSL 3.0.
```

Comment:
This was added in late september.. (Just after 3.0) so I thought it was in that branch... I cant see it in 3.0.7 so you must be right..